### ISSUE TITLE
How can i specify published messages in a loop by connected session info in one realm?

### ISSUE THREAD

**Issue by WebdevMerlion** [voryx/Thruway#202]
Hi! So, i have one PHP client, that push messages to a router in a loop. Also have a lot of Autobahn (JS) clients thats subscripts on it, which authenticated with wamp-cra.

How can i specified pushed info up on authid? Or this is not possible in one realm?

Tried to start loop up on RPC call, where i can get session info from connected client, but this is not asynch and it publish whole information for all connected clients about all connected clients

On the server side Client:
```
$onProvide = function($args, $kwargs, $details) use ($loop, $session, $stmt, &$timer){
        $client = $details->authid;

        $timer = $loop->addPeriodicTimer(
            1,
            function () use ($session, $stmt, $client) {
                $counter = getInfo($client, $stmt);
                $session->publish('com.myapp.push', [$counter]);
            }
        );
    };

$session->register('com.myapp.provide', $onProvide, ['disclose_caller' => true]);
```

To put it simply, i want to split pushed info from the pusher by joined session info. Oublisher and listeners are joined to one realm.

Sorry, if i missed some related issues.

**Comment by davidwdan** [comment]
You can publish to individual sessions with the publish option `'eligible'    => $sessionIds` and to individual authIds with `'_thruway_eligible_authroles'    => $authIds`.  
`_thruway_eligible_authroles` only works with the Thruway router.

**Comment by WebdevMerlion** [comment]
Thanks! I split info exaсtly as i need! But every new connected session start new timer, and it increase interval for all sessions. Is it possible to asynch loop for sessions?

**Comment by WebdevMerlion** [comment]
Even more, how can i update info, which will be used inside timer callback in a next periodic timer tick?

**Comment by mbonneau** [comment]
@WebdevMerlion - Did you still need help with this?

**Comment by WebdevMerlion** [comment]
No, thanks. I got it.
